src/request/sign-transaction/SignTransaction.js

```diff
@@ -1,4 +1,4 @@
-class SignTransactionView extends Nimiq.Observable { // eslint-disable-line no-unused-vars
```

Comment:
Why are you even changing `sign-transaction` files in a PR about the `create` request? Should be moved out into a separate PR.